### PR TITLE
blur bg on err pages

### LINKED ISSUE #1485: Blur background on all but Swap pages
Minor thing i missed:
- The profile container misses this setting `backdrop-filter: blur(50px);`

Otherwise OK to go as is.

_Originally posted by @biocom in https://github.com/gnosis/cowswap/issues/1376#issuecomment-928944645_

### PR DESCRIPTION
**Pull request by maria-vslvn**
# Summary

Fixes #1485 

reproduce an err page an check the background

  # To Test

1. Open the page[ "Cow Game" ](https://pr1829--gpswapui.review.gnosisdev.com/#/play/cow-runner)to throw an error so we can see the app crashing.

image

### CODE REVIEW

**Review by alongoni — CHANGES_REQUESTED** [review]
Looks great, but we should add the Cowswap logo to give users an option to go back to home.

I've worked on this solution some time ago. Please check the [PR#1259](https://github.com/gnosis/cowswap/pull/1259). And especially the way we include the logo here:
https://github.com/gnosis/cowswap/pull/1259/commits/a1d52105b3a6b16c737506259ff8d165ed8f97ee

**Comment by elena-zh** [comment]
Hi @maria-vslvn , how can I generate a hard error from the UI part in order to test the changes?

**Comment by maria-vslvn** [comment]
@elena-zh @alongoni some changes are pushed, now the bug can be reproduced by trying to play the runner-cow game

**Review by elena-zh — CHANGES_REQUESTED** [review]
Changes LGTM besides one issue: I can't scroll down the page to see links at the bottom of the error section
(I was able to reproduce the issue in Firefox)
image

**Comment by anxolin** [comment]
The change looks good, but it also happens to me the issue with the scroll

**Comment by alongoni** [comment]
Hey @maria-vslvn I've found this: (it's a class called `noScroll`)
image
image

**Comment by maria-vslvn** [comment]
@elena-zh @W3stside i added footer, and removed the noScroll in the view, previously i asked for help in fixing the script, that  existed there and added the class for all views, so i can close it if there is no need in this?

**Comment by elena-zh** [comment]
Hey @maria-vslvn , changes LGTM! However, it would be great to update the branch with the latest version 1.9.0, due to 1.4.1 is here

**Comment by elena-zh** [comment]
Changes LGTM. 
The only issue I can mention is that the page has become wider than usual content pages
image
 Compare with 'About' page
image


@alfetopito , are we OK with this?

File: src/custom/pages/games/CowRunner/index.tsx [hunk 1]
```diff
@@ -29,6 +29,10 @@ const Wrapper = styled(Page)`
 `
 
 export default function CowRunnerPage() {
+  if (0 === 0) {
+    throw new Error('This cow is dropping some serious milk 🥛!!')
+  }
+
```
Comment (alfetopito) [inline src/custom/pages/games/CowRunner/index.tsx:35]: Please remember to remove this before merging!

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
@maria-vslvn , could you please revert the change that forces a hard error on the CowGame page? 
Then we can merge this PR into Develop

**Review by elena-zh — APPROVED** [review]

**Comment by elena-zh** [comment]
@maria-vslvn . @ramirotw , could we please merge this?

**Pull request merged**